### PR TITLE
#362 countdown page and drawer fix

### LINKED ISSUE #362: Countdown page and drawer fix
### **Describe the bug**
The countdown page has design bugs for smaller screens and also users can still access the initial pages deployed. We also need to update the drawer and group menu links into appropriate sections

### **To Reproduce**
1. Go to https://materialsmine.org/explorer. That page should not be accessible.
2. Go to https://materialsmine.org. View on mobile or tablet screens and you'll notice the design inconsistencies.

### **Expected behavior**
Fix design bugs and update all routes to point to the new countdown page.

### PR DESCRIPTION
**Pull request by tholulomo**
Fix the countdown page design bugs for smaller screens and also issues where users can still access the initial pages deployed on prod. 

Updated the drawer and group menu links into appropriate sections.

closes #362

### CODE REVIEW

**Comment by aswallace** [comment]
Noticing a slight misalignment for smaller screen sizes (I think it's tab-port, tab-land, or tablet, but can't quite tell)

Screen Shot 2023-03-22 at 10 44 47 AM

File: app/src/components/Drawer.vue [hunk 1]
```diff
@@ -92,33 +77,46 @@
...
       </md-list-item>
+    </md-list>
+
+    <md-list class="utility-transparentbg">
       <md-list-item v-if="isAuth" @click="$store.dispatch('auth/logout')">
         <md-icon class="utility-navfonticon">logout</md-icon>
         <span class="md-list-item-text utility-navfont">Log out</span>
       </md-list-item>
-      <md-list-item v-if="!isAuth" @click="login">
+      <md-list-item class="" v-if="!isAuth" @click="login">
```
Comment (aswallace) [inline app/src/components/Drawer.vue:114]: small nit: does `class=""` need to be here?

**Review by aswallace — APPROVED** [review]
Minor comment about alignment and a couple other non-essential suggestions. Also, I still think that transparent email field is a little hard to read

File: app/src/components/Drawer.vue [hunk 2]
```diff
@@ -92,33 +77,46 @@
...
+
+      <md-toolbar class="md-transparent u--font-emph-l u_margin-top-small" md-elevation="0" v-if="isAuth" ><small>User Dashboard</small></md-toolbar>
+      <md-divider v-if="isAuth" ></md-divider>
+      <md-list-item md-expand v-if="isAuth">
+        <md-icon class="utility-navfonticon">upload</md-icon>
+        <span class="md-list-item-text utility-navfont">Curate</span>
         <md-list slot="md-expand">
-          <router-link :to="'/nm/contact-inquiry'" v-slot="{navigate, href}" custom>
-            <md-list-item :href="href" @click="navigate"  class="md-inset">Contact Inquiry</md-list-item>
+          <!-- <md-list-item class="md-inset" href="https://materialsmine.org/nm#/XMLCONV">
```
[the remark this answers is not in the document]
Comment (tholulomo) [reply app/src/components/Drawer.vue:87]: Yes, but we haven't merged that code in yet. I'm trying to remove redundant links, but this stood out to me

**Pull request merged**